src/app/login-dialog/login-dialog.component.html

```diff
@@ -0,0 +1,83 @@
+<!--Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.-->
+
+<!--Licensed under the Apache License, Version 2.0 (the "License");-->
+<!--you may not use this file except in compliance with the License.-->
+<!--You may obtain a copy of the License at-->
+
+<!--http://www.apache.org/licenses/LICENSE-2.0-->
+
+<!--Unless required by applicable law or agreed to in writing, software-->
+<!--distributed under the License is distributed on an "AS IS" BASIS,-->
+<!--WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.-->
+<!--See the License for the specific language governing permissions and-->
+<!--limitations under the License.-->
+
+<div class="md-hue-2 login-bg"
+     md-colors="{background: 'default-primary-700'}"
+     fxLayout="column"
+     fxLayoutAlign="center center"
+     flex>
+  <mat-dialog-content>
+    <mat-card align="end">
+      <i class="material-icons close-modal" (click)="close()">
```

Comment:
is it ok to add `(click)` to an `i` element? Does it change the cursor to a pointer? If no, then you may want to wrap it inside an `a` element